src/snmp_core.h

```diff
@@ -11,11 +11,14 @@
 #ifndef SQUID_SNMP_CORE_H
 #define SQUID_SNMP_CORE_H
 
+#if SQUID_SNMP
```

Comment:
I hope this is no longer needed (but you will need to remove the matching `#endif` as well):

```suggestion
```

Reply:
it isn't. removing.